browser/ui/views/profiles/brave_incognito_menu_view.cc

```diff
@@ -38,20 +38,15 @@ void BraveIncognitoMenuView::ButtonPressed(views::Button* sender,
   }
 }
 
-void BraveIncognitoMenuView::AddTorButton(
-    ProfileMenuViewBase::MenuItems* menu_items) {
+void BraveIncognitoMenuView::AddTorButton() {
```

Comment:
cc @darkdh we have identical logic here and https://github.com/brave/brave-core/pull/2892/files#diff-076f833c0d691c629974c9447f66b39cR49 that should be refactored into a helper method or something as a follow-up

Reply:
addressed in cd39beb7a59ab40fdf185b0b2c39ca270d45b51e